### PR TITLE
Bug actions

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

File: src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  For help and questions around Keycloak please see [keycloak.org/community](https://www.keycloak.org/community).
+
+action-MISSING_INFO=\
+  Thanks for reporting this issue, but there is insufficient information or lack of steps to reproduce.\n\
+  \n\
+  Please provide additional details, otherwise the issue will be automatically closed within 14 days.
+
+action-OLD_RELEASE=\
+  Thanks for reporting this issue, but as this is reported against an older and unsupported release we are not able \
+  to evaluate the issue. Please verify with the latest release and update the issue if the issue still exists.\n\
```
Comment (ssilvert) [inline src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties:13]:
```suggestion
  to evaluate the issue. Please verify with the [nightly build](https://github.com/keycloak/keycloak/releases/tag/nightly) if possible, or [latest release](https://www.keycloak.org/downloads) at a minimum.  Then update the issue if the issue still exists.\n\
```

**Review by ssilvert — CHANGES_REQUESTED** [review]

File: src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties (hunk 1 above)
Comment (stianst) [reply src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties]: Update the message to reflect your suggestion

File: src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+action-MISSING_INFO=\
+  Thanks for reporting this issue, but there is insufficient information or lack of steps to reproduce.\n\
+  \n\
+  Please provide additional details, otherwise this issue will be automatically closed within 14 days.
+
+action-OLD_RELEASE=\
+  Thanks for reporting this issue, but as this is reported against an older and unsupported release we are not able \
+  to evaluate the issue. Please verify with the [nightly build](https://github.com/keycloak/keycloak/releases/tag/nightly)\
+  or the [latest release](https://www.keycloak.org/downloads).\n\
```
[the remark this answers is not in the document]
Comment (stianst) [reply src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties:14]: We should not require folks to use the nightly, and they should be able to report issues against any supported release.

File: src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+action-INVALID=\
+  Thanks for reporting this issue. However, after review this is not considered a valid issue, or has been recently resolved.\n\
+  \n\
+  As the issue is not valid it will be automatically closed.
+
+action-PRIORITY_NORMAL=\
+  Due to the amount of issues reported by the community we are not able to prioritise resolving this issue at the moment.\n\
+  \n\
+  If you are affected by this issue, upvote it by adding a :thumbsup: to the description. We would also welcome a \
+  contribution to fix the issue.
```
[the remark this answers is not in the document]
Comment (stianst) [reply src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties:28]: Messages shouldn't include the clean-up part. We may decide to change the number of days, and we may close issues at any given time if there's no updates.

File: src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+action-PRIORITY_LOW=\
+  Due to the amount of issues reported by the community we are not able to prioritise resolving this issue at the moment.\n\
+  \n\
+  If you are affected by this issue, upvote it by adding a :thumbsup: to the description. We would also welcome a \
+  contribution to fix the issue.
```
[the remark this answers is not in the document]
Comment (stianst) [reply src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties:34]: Messages shouldn't include the clean-up part. We may decide to change the number of days, and we may close issues at any given time if there's no updates.

**Pull request merged**